docs/source/_templates/layout.html

```diff
@@ -23,5 +23,17 @@
   ga('send', 'pageview');
 
 </script>
+
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-117752657-2"></script>
+
+<script>
+  window.dataLayer = window.dataLayer || [];
+
+  function gtag(){dataLayer.push(arguments);}
+
+  gtag('js', new Date());
+  gtag('config', 'UA-117752657-2');
```

Comment:
What is the UA for? It is different for the ones I have for PyTorch.

Reply:
It's a Global Site Tag that @brucejlin asked me to add to the static, docs, and tutorial sites.